wiki/redirect.yaml

```diff
@@ -465,8 +465,8 @@
 "tips_and_tricks_on_skill_improvement":                       "Guides/Tips_and_Tricks_on_Skill_Improvement"
 "videos_from_youTube":                                        "Guides/Videos_From_YouTube"
 
-"help_center":                 "Help_Center"
-"help":                        "Help_Center"
+"help_centre":                 "Help_Centre"
```

Comment:
add the other varient too (so other links and American English people can go to the right location)